packages/bun-types/net.d.ts

```diff
@@ -528,6 +528,7 @@ declare module "net" {
    * @since v0.1.90
    */
   class Server extends EventEmitter {
+    listening: boolean;
```

Comment:
on nodejs this is a getter, so you can do something like:
```suggestion
    listening: boolean;
    get listening() {
        return !!this.#server;
    }
```

Reply:
This is now defined in `/@types/node/net.d.ts` where it's actually not marked as readonly, but that could be an upstream contribution then.